app/views/roles/_form.html.erb

```diff
@@ -12,7 +12,26 @@
     <div class="tab-pane active" id="primary">
       <%= text_f f, :name, :class => @role.builtin? ? "disabled" : ""  %>
       <%= hidden_field_tag :original_role_id, @original_role_id if @cloned_role %>
+
+      <% if show_location_tab? || show_organization_tab? %>
+        <%= alert(:close => false, :class => 'alert-info', :header => '', :text => _("Changes to following associations will propagate to all inheriting filters ") +
+            popover("", _("Whenever organizations or locations role association is changed, the change will propagate to all inheriting filters.
+                               Filters that are set to override will remain untouched. Overriding of role filters can be easily disabled by
```

Comment:
"can be easily disabled by [..]" => "can be easily disabled by pressing the "Disable overriding" button"?